test/test_prompt_generators.py

```diff
@@ -4,6 +4,8 @@
 from skimage.data import binary_blobs
 from skimage.measure import label
 
+COLOR_CYCLE = ["#00FF00", "#FF0000"]
```

Comment:
This shouldn't be added here.

Reply:
Ah okay. I would just pass it directly to the function then rather globally as now. Thanks.